_posts/2020-09-17-using-tettra.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Using Tettra to Keep our Content Organized"
+date: 2020-09-17 11:00:00
+reviewed: 2020-09-17 11:00:00
+categories: ["team", "remote"]
+author: "abizzinotto"
+---
+
+At OmbuLabs we use [Tettra](https://app.tettra.co) to power our knowledge base. As a remote, distributed team it is very important to us to have our content structured and organized. We follow some good practices to keep it organized and up to date, which we describe [here](link to kb article), and we use Tettra to keep our knowledge organized and integrated with the tools we use on a daily basis.
```

Comment:
The landing page has probably more details
```suggestion
At OmbuLabs we use [Tettra](https://tettra.com/) to power our knowledge base. As a remote, distributed team it is very important to us to have our content structured and organized. We follow some good practices to keep it organized and up to date, which we describe [here](link to kb article), and we use Tettra to keep our knowledge organized and integrated with the tools we use on a daily basis.
```

Reply:
Also, you're missing a link in: `[here](link to kb article)`

Reply:
> Also, you're missing a link in: `[here](link to kb article)`

@ABizzinotto ping

we should remove that until the other article is published.